### ISSUE TITLE
DEFAULT_GRAPH_VERSION is deprecated. v3.0 should be used

### ISSUE THREAD

**Issue by thoford75** [facebookarchive/php-graph-sdk#1044]
https://github.com/facebook/php-graph-sdk/blob/3138249d85e8dc040f15bedd3581a3c616f49005/src/Facebook/Facebook.php#L61

**Comment by SammyK** [comment]
Yeah, we've been bumping this periodically even though it's a BC break but stopped doing it at some point. @yguedidi - Are we still planning on bumping this again? It's really only a BC break for those that don't listen to the recommendation of including the `default_graph_version` in their config. :)

**Comment by yguedidi** [comment]
@SammyK what I'm asking myself is should we upgrade to the highedt available version, or the lowest supported one?
